examples/ClientTests/CMakeLists.txt

```diff
@@ -632,4 +632,18 @@ if (DD4HEP_USE_GEANT4)
       REGEX_PASS "ResourcesAfterConstruction       ConstructSD:   VmRSS"
       REGEX_FAIL "Error;ERROR; Exception"
   )
+
+  dd4hep_add_test_reg(ClientTests_ddsim_setup_BoxOfStraws
```

Comment:
The test should only run when DDG4 and LCIO are used for building.
Also currently gets no hits. Not sure if shooting in a bad direction, or SD not working.